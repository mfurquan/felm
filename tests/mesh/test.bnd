1
LN02
2	4	T	11.
