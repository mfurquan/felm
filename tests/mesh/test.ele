2
TR03
2	3	4
QR04
1	2	4	5
