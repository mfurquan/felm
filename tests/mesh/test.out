 Elements:
 TR03           2           3           4
 QR04           1           2           4           5
 Nodes:
   0.0000000000000000        0.0000000000000000        0.0000000000000000     
   1.0000000000000000        0.0000000000000000        0.0000000000000000     
   2.0000000000000000       0.50000000000000000        0.0000000000000000     
   1.0000000000000000        1.0000000000000000        0.0000000000000000     
   0.0000000000000000        1.0000000000000000        0.0000000000000000     
 Boundary faces:
 LN02           2           4   11.000000000000000      T
